src/Draco.Compiler/Internal/Declarations/DeclarationTable.cs

```diff
@@ -90,17 +94,26 @@ private MergedModuleDeclaration BuildMergedRoot()
                     path, this.RootPath));
 
                 // Add to root so the compilation can continue
-                modules.Add(new SingleModuleDeclaration(rootName, rootNameSplitPath, (CompilationUnitSyntax)tree.Root));
+                modules.Add(new SingleModuleDeclaration(
+                    name: rootPath.Last,
+                    path: rootPath.Slice(^1..),
```

Comment:
Why not use pathBeforeRoot?

Reply:
here and in all the other places

Reply:
`pathBeforeRoot` is everything before the root module. So if the root in `foo/bar/baz`, then `pathBeforeRoot` is `foo/bar` and `rootPath.Slice(^1..)` is `baz`